.mailmap

```diff
@@ -0,0 +1,11 @@
+Ares Meroueh <ares.m@asidatascience.com> ameroueh <ares.meroueh@gmail.com>
```

Comment:
```suggestion
Ares Meroueh <ares.m@faculty.ai> ameroueh <ares.meroueh@gmail.com>
```

Reply:
Can do. This will also require an additional line that maps `Ares Meroueh <ares.m@faculty.ai>` to `Ares Meroueh <ares.m@asidatascience.com>`.